<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>901</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>901</width>
    <height>301</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_filePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SelectFile">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>浏览…</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_VerifyKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>在此输入对比Key值</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_SelectCoder">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>MD5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SHA-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SHA-256</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Verify">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>校验</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>861</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="pushButton_Exit">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/mainwindow/icon/Res/UnversionedIcon.ico</normaloff>:/mainwindow/icon/Res/UnversionedIcon.ico</iconset>
   </property>
   <property name="text">
    <string>关于此软件(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于此软件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_filePath</tabstop>
  <tabstop>pushButton_SelectFile</tabstop>
  <tabstop>lineEdit_Key</tabstop>
  <tabstop>comboBox_SelectCoder</tabstop>
  <tabstop>lineEdit_VerifyKey</tabstop>
  <tabstop>pushButton_Verify</tabstop>
  <tabstop>pushButton_Exit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
